catalog/app/containers/Bucket/PackageTree.js

```diff
@@ -84,6 +88,15 @@ function RevisionInfo({ revisionData, revision, bucket, name, path }) {
     _: R.identity,
   })
 
+  const getHttpsUri = (r) =>
+    `${window.origin}${urls.bucketPackageTree(bucket, name, r.hash, path)}`
```

Comment:
Hm... I expected `quilt+s3://` url here

Reply:
Now it's the same URL as if you copy it by right click and select "Copy link location"

![Screenshot from 2020-12-17 11-34-40](https://user-images.githubusercontent.com/533229/102463250-e863fa80-405b-11eb-9e33-ea35d58d44d7.png)